primitive/uuid.go

```diff
@@ -27,6 +27,11 @@ const LengthOfUuid = 16
 
 type UUID [16]byte
 
+func (u UUID) Clone() UUID {
```

Comment:
As I said earlier, it'd be more flexible to have this method on the point `*UUID` type, returning `*UUID`.